.circleci/config.yml

```diff
@@ -1,7 +1,321 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER - Login user for docker hub
+# DOCKER_PASS - Login password for docker hub user
```

Comment:
So when the jobs say `Warning: No authentication provided, using CircleCI credentials for pulls from Docker Hub.`, it means they are using our credentials, but that they were set up in CircleCI ?

Reply:
These credentials are used for pushing the generated docker images in the `release-*` jobs. We don't need to provide our own credentials for pulling because Circle is allowed to pull lots of images by Docker.